package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "commercetools-nodejs-boilerplate",
+  "version": "1.0.0-beta.1",
+  "description": "Generic Boilerplate for bootstrapping NodeJS applications",
+  "keywords": [
+    "commercetools",
+    "nodejs",
+    "boilerplate",
+    "starter"
+  ],
+  "homepage": "https://github.com/commercetools/nodejs-boilerplate#readme",
+  "bugs": {
+    "url": "https://github.com/commercetools/nodejs-boilerplate/issues"
+  },
+  "license": "MIT",
+  "contributors": [
+    "Abimbola Idowu <abimbola.idowu@commercetools.de> (https://github.com/hisabimbola)",
+    "Nicola Molinari <nicola.molinari@commercetools.de>",
+    "Philipp Sporrer <philipp.sporrer@commercetools.de>",
+    "Selwyn <talk@selwyn.cc> (https://selwyn.cc/)"
+  ],
+  "main": "./lib/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs-boilerplate.git"
+  },
+  "scripts": {
+    "start": "npm run watch",
+    "test": "cross-env NODE_ENV=test babel-node test/*.spec.js test/*/**spec.js | tap-spec",
+    "test:watch": "watch \"npm run coverage:html\" src test -d",
```

Comment:
But why is it under `test:watch` ?

We usually use this task in development, specifically when writing tests. I don't want to run the coverage at this stage. 

I'm also not sure about the `watch` task, if it's really necessary. We should keep one or the other.
And usually coverage is something a build service runs.

What do you guys think?